### PR TITLE
Add v0.20.0 and v0.20.1

### PR DESCRIPTION
**Pull request by stevenroose**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -10,6 +10,7 @@ env:
   - BITCOINVERSION=0.19.0.1
   - BITCOINVERSION=0.19.1
   - BITCOINVERSION=0.20.0
+  - BITCOINVERSION=0.20.1
```
Comment (RCasatta) [inline .travis.yml:12]: we may consider to only support last minor version of every major version? like: 0.20.1 0.19.1, 0.18.1 ?

**Review by RCasatta — APPROVED** [review]
the comment I left is not blocking

ack 897aeb42d4e3d02f263af222f1fff9a5c4311f28

File: .travis.yml (hunk 1 above)
Comment (stevenroose) [reply .travis.yml:12]: Just to reduce the number of Travis Jobs, you mean? Perhaps we can just not have a matrix with all the Rust versions as well. And just do the integration test with stable and then test building (and unit tests) with all Rust versions.

Comment (RCasatta) [reply .travis.yml:12]: Just keep in mind one of my projects has been marked for misuse for fewer jobs. No way to contact travis support, I migrate to github action only for that reason...
minor version upgrade in bitcoin should not break rpc interface and usually they are bug fixing so not worth testing the previous minor version imo

**Pull request merged**